src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from 'react-router-dom';
 
 const App = () => (
   <div>
     <Navigation />
+    <Route path="/" component={Home} />
```

Comment:
one thing missing: the `exact` prop
as written, your <Home /> component is rendering along with /about and /contact

```
<Route exact path="/" component={Home} />
```